packages/edit-site/src/components/editor/index.js

```diff
@@ -94,7 +94,12 @@ function Editor() {
 		};
 	}, [] );
 	const { updateEditorSettings } = useDispatch( 'core/editor' );
-	const { setPage, setIsInserterOpened } = useDispatch( editSiteStore );
+	const { setPage, setIsInserterOpened, updateSettings } = useDispatch(
+		editSiteStore
+	);
+	useEffect( () => {
+		updateSettings( initialSettings );
+	}, [ initialSettings ] );
```

Comment:
Why have `initialSettings` as dependency? Shouldn't an empty array to run once be enough?

Reply:
It should be enough 🤔 Let's give it a try